libsolidity/ast/AST.h

```diff
@@ -2182,6 +2195,7 @@ class FunctionCall: public Expression
 	std::vector<ASTPointer<Expression>> m_arguments;
 	std::vector<ASTPointer<ASTString>> m_names;
 	std::vector<SourceLocation> m_nameLocations;
+	bool m_isSuffixCall;
```

Comment:
similar as above.. as @ekpyron wrote, it cannot hurt to always initialize

Reply:
Except this one :-)?

Reply:
Ah, my bad. I did not look closely enough to see that it's about a different class. Fixed now.